src/styles/main.scss

```diff
@@ -1,3 +1,203 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
```

Comment:
redundant reset margin and padding for all selectors. reset them only for elements which need it
```suggestion
```